### PR TITLE
Don't clear cml output and lazy load browser test server

### PR DESCRIPTION
**Pull request by shanemc-goog**
### Description

For browser test setup

1. Stop clearing the output to the console as it hides compile errors.
2. Add a lazy load loop to fully initialize the server sooner.

## Release notes:

NA testing only

#### General

- [ ] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/application.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: bin/run-browser-test-env [hunk 1]
```diff
@@ -142,5 +142,19 @@ echo "Running docker compose to start the CiviForm environment:"
 if truth::is_enabled continuous_integration; then
   compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate >.dockerlogs 2>&1 &
 else
-  compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate
+  compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate --wait -d
+fi
+echo "Running now, starting load loop"
```
Comment (shanemc-goog) [inline bin/run-browser-test-env:147]:
```suggestion
```

File: bin/run-browser-test-env [hunk 2]
```diff
@@ -142,5 +142,18 @@ echo "Running docker compose to start the CiviForm environment:"
 if truth::is_enabled continuous_integration; then
   compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate >.dockerlogs 2>&1 &
 else
-  compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate
+  compose_browser_test "${use_dev}" --profile ${cloud_provider} up --no-recreate --wait -d
+fi
+
+# Start a warmup request loop so that when the subsequent server startup is ready, we trigger lazy
+# loading its edit-refresh resources so save developer time.
```
Comment (shanemc-goog) [inline bin/run-browser-test-env:149]:
```suggestion
# loading its edit-refresh resources to save developer time.
```

**Comment by nbeloglazov** [comment]
We still need to keep similar polling loop in run-browser-test script? Because without it tests will start before server becomes healthy?

**Comment by shanemc-goog** [comment]
> We still need to keep similar polling loop in run-browser-test script? Because without it tests will start before server becomes healthy?

This works the same as bin/run-dev

1. the server starts in lazy-load mode
2. Something makes a request of the server, be it the dev loading the site or `bin/run-browser-tests` accessing the server
3. The server start lazy-load compiling.

So same as bin/run-dev the loop will start the lazy load compile sooner.

File: bin/run-browser-test-env [hunk 3]
```diff
@@ -142,5 +142,18 @@ echo "Running docker compose to start the CiviForm environment:"
...
+fi
+
+# Start a warmup request loop so that when the subsequent server startup is ready, we trigger lazy
+# loading its edit-refresh resources to save developer time.
+for i in {1..60}; do
+  sleep 5
+  if curl --output /dev/null --silent --fail http://localhost:9999/loginForm; then
+    break
+  fi
+done &
```
Comment (gwendolyngoetz) [inline bin/run-browser-test-env:155]: What's the reason for making the loop a background process?

Comment (gwendolyngoetz) [reply bin/run-browser-test-env:155]: Ahh

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**